config/locales/de.yml

```diff
@@ -341,6 +342,7 @@ de:
         one: 'Konnte %{resource} nicht speichern: ein Fehler.'
         other: 'Konnte %{resource} nicht speichern: %{count} Fehler.'
   entry_year: Einreisejahr
+  find_client: KlientIn finden
```

Comment:
we just took over the way aoz uses it, that we don't have to discuss personal/political topics 😄